Update multi-echo.rst

handwerkerd opened the pull request:
<!---
This is a suggested pull request template for tedana.
It's designed to capture information we've found to be useful in reviewing pull requests.

If there is other information that would be helpful to include, please don't hesitate to add it!

Please also label your pull request with the relevant tags.
See here for more information and a list of available options:
https://github.com/ME-ICA/tedana/blob/master/CONTRIBUTING.md#pull-requests
-->

<!-- Please indicate after the # which issue you're closing with this PR.
This is helpful for the maintainers AND will magically close the issue when this
pull request is merged!
https://help.github.com/articles/closing-issues-using-keywords -->
Progress on #83 .

<!-- Please give a brief overview of what has changed in the PR.
If you're not sure what to write, consider it a note to the maintainers to indicate
what they should be looking for when they review the pull request. -->
Changes proposed in this pull request:

- Adds several paragraphs on acquisition recommendations to the documentation
- The part of #83 that is still missing is filling out the "Sequences" section of the documentation to say the status/location of multi-echo sequences for each of the main vendors.
- I'm hesitant to add specific scanning parameters here because I don't want to recommend anything overly-specific that users might thoughtlessly copy and use, unless those specific parameter sets have been validated in some way (or at least could be linked to specific publications)

emdupre commented on docs/multi-echo.rst:
```diff
@@ -48,6 +48,16 @@ Sequences
 
+Acquisition Parameter Recommendations
+************************************
+There is no empirically tested best parameter set for multi-echo acquisition. The guidelines for optimizing parameters are similar to single-echo fMRI. Parameter choices require balancing the needs of specific studies with regards to spatial resolution and coverage, sampling rate, signal-to-noise ratio, signal drop-out, distortion, and artifacts. Optimize these parameters based on whatever a stud ...
```
Should we say _why_ there is this compromise ? Or point out to a reference ?

emdupre commented on docs/multi-echo.rst:
```diff
@@ -48,6 +48,16 @@ Sequences
+************************************
+There is no empirically tested best parameter set for multi-echo acquisition. The guidelines for optimizing parameters are similar to single-echo fMRI. Parameter choices require balancing the needs of specific studies with regards to spatial resolution and coverage, sampling rate, signal-to-noise ratio, signal drop-out, distortion, and artifacts. Optimize these parameters based on whatever a stud ...
+
+A minimum of 3 echoes is recommended for running TE-dependent denoising. More echoes may be useful, because that would allow for more accurate estimates of BOLD and non-BOLD weighted fluctuations, but more echoes would require additional compromises in spatiotemporal coverage or more acceleration. Whether the benefits of more echoes balance out the additional acquisition compromises is an open re ...
```
Should we say the general recommendation for a single-echo T2* weighted fMRI is a TE of about 30ms ?

handwerkerd commented:
I re-wrote the text to address the above comments. I've added more line breaks. I added a more detailed explanation of the cost of multi-echo and how that relates to other parameter choices. As part of the explanation, I included an example TE of 30s. I don't want to mention 30ms out of an example because that is a 3T specific parameter and I didn't want to hunt for a good reference on the TE to T2* relationship across field strengths and tissue types right now. The edited version is on the PR at https://github.com/ME-ICA/tedana/blob/pr/282/docs/multi-echo.rst but I'm not sure why I can't seem to link to it directly on this PR page.

jbteves commented:
@handwerkerd did you push your commit to this branch? For some reason the PR only shows the original commit.

jbteves commented:
Also, @handwerkerd did you mean to fork from your master branch? If you do a squash merge into `tedana` that will make your history a bit hairy.

emdupre commented:
I think it was probably an accident, but it's one we can fix pretty easily ! One idea would be to just close this PR, and then open a new PR from the "pr/228" branch (https://github.com/ME-ICA/tedana/blob/pr/282/docs/multi-echo.rst).

To "fix" your master branch, then just `git fetch upstream` and `git merge master` after we merge that PR in.

Let me know if that seems ok !

emdupre commented:
PS: I _really_ like the updated text, @handwerkerd ! Thanks again :sparkles:
